.gitignore

```diff
@@ -8,3 +8,4 @@ platformio-device-monitor*.log
 logs/device-monitor*.log
 platformio_override.ini
 .DS_Store
+platformio_override.ini
```

Comment:
Why do you add the file a second time?

Reply:
I did not expect the file to be mentioned in `.gitignore` already, I put it in there withouth checking whether or not it was already in there... Thanks for noticing!